im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
+    <link href="http://www.zotero.org/styles/im-gespraech-buber-gesellschaft" rel="self"/>
+    <author>
```

Comment:
![aa nachweis-richtlinien ig allgemein 01 2013_s1](https://cloud.githubusercontent.com/assets/14075893/9620184/eb7253ea-5117-11e5-9acf-07ad4334054d.jpg)
![aa nachweis-richtlinien ig allgemein 01 2013_s2](https://cloud.githubusercontent.com/assets/14075893/9620186/f3a6d0ae-5117-11e5-83b8-9423b320e093.jpg)

This "manual" is from 2013, at the moment we are working on a better manual but there are in the rules themselves only very small changes.